### DISCUSSION TITLE
GitHub Contribution History goes back to 1969

### DISCUSSION THREAD

**Discussion by elderguardian** [community/community#72614] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I tried creating art using the contribution graph. After deleting the commits and the repository, my profile's contribution history still dates back to the year when I initially experimented with it. On my profile, every year since 1969 is now displayed. I waited to see if it was a caching issue, but it doesn't appear to be.

![image](https://github.com/community/community/assets/129489839/2db79230-7e2e-4021-97f6-4b6eed678539)

![image](https://github.com/community/community/assets/129489839/0d25a17c-4b05-4e99-b9ae-123f6ff83624)

**Comment by Treata11** [comment]
It's kinda cool you know
How can I include these years on my profile? :)

**Reply by elderguardian** [reply]
You need a repository that contains commits created years ago and are bound to your email in your profile. Create a new repository on GitHub, import your local repository with the old commits and delete the repository afterwards.

**Reply by Treata11** [reply]
Based on your explanations, if you've deleted the repo that _contained_ the commits dating back to 1969, there nothing you can do currently since the delete action of the repo can't be **undone** to rewrite the commit history.
But if the repo is not deleted yet, just change the commits' dates & **force** push 'em & you probably be alright.

**Comment by adityaongit** [comment]
How did you solve this bug though? I am having the same issue.

**Reply by Treata11** [reply]
There's no fix for it.

**Reply by PerFrivik** [reply]
gg I have the same issue, I was working on a robot and the timesync was messed up, now apparently I have been coding since 1969 :laughing:

**Comment by manish-lal12** [comment]
I had been facing the same issue, I was able to fix in following ways:
1) Check and prune commits made on old dates (e.g., 1970s). Can use `git filter-repo` tool to rewrite Git repository history, or delete repository if irrelevant.
2) Push the updated commits using `git push --force --all origin`.
3) If still old years(1970s) show up in timeline which was the case for me, update the email in github profile, it triggers a refresh and rebuild of contribution timeline cache which I figured out after contacting Github support. 

Hopefully, this fixes the bug for anyone who’s been stuck with this frustrating issue for a long time! 😌

**Reply by jasamandehvary** [reply]
Thanks, this solution is working!
